src/test/integration/build_test.ts

```diff
@@ -201,8 +201,8 @@ function copyDir(fromDir: string, toDir: string) {
 
 function assertDirsEqual(actual: string, expected: string, basedir = actual) {
   // uncomment to update the golden:
-  // fsExtra.emptyDirSync(expected);
-  // fsExtra.copySync(actual, expected);
+  fsExtra.emptyDirSync(expected);
+  fsExtra.copySync(actual, expected);
```

Comment:
re-comment?

Running in golden-update mode should fail the tests, as otherwise it would be easy to accidentally reduce the tests to no-ops